native/acter/src/api/verification.rs

```diff
@@ -120,7 +120,11 @@ impl VerificationEvent {
                         let sequence = items
                             .iter()
                             .map(|e| {
-                                let chr = e.symbol.chars().next().expect("first char not found");
+                                let chr = e
+                                    .symbol
+                                    .chars()
+                                    .next()
+                                    .expect("At least one character should be present");
```

Comment:
we are confident this crashing the app is a good choice?!? Shouldn't we instead have a `filter_map(` here and instead return `None` in cases we can't read the char? Probably return a broken emoji list and the user can then cancel the process..

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1252/commits/577ad8847d232d969c8478ff9b59311427486f3b